specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/networkWatcher.json

```diff
@@ -4152,13 +4168,61 @@
         "filter": {
           "$ref": "#/definitions/ConnectionMonitorEndpointFilter",
           "description": "Filter for sub-items within the endpoint."
+        },
+        "scope": {
+          "$ref": "#/definitions/ConnectionMonitorEndpointScope",
+          "description": "Endpoint scope."
+        },
+        "coverageLevel": {
+          "type": "string",
+          "enum": [
+            "Default",
+            "Low",
+            "BelowAverage",
+            "Average",
+            "AboveAverage",
+            "Full"
+          ],
+          "x-ms-enum": {
+            "name": "CoverageLevel",
+            "modelAsString": true
+          },
+          "description": "Test coverage for the endpoint."
         }
       },
       "required": [
         "name"
       ],
       "description": "Describes the connection monitor endpoint."
     },
+    "ConnectionMonitorEndpointScope": {
+      "properties": {
+        "include": {
+          "type": "array",
+          "description": "List of items which needs to be included to the endpoint scope.",
+          "items": {
+            "$ref": "#/definitions/ConnectionMonitorEndpointScopeItem"
+          }
+        },
+        "exclude": {
+          "type": "array",
+          "description": "List of items which needs to be excluded from the endpoint scope.",
+          "items": {
+            "$ref": "#/definitions/ConnectionMonitorEndpointScopeItem"
+          }
+        }
+      },
+      "description": "Describes the connection monitor endpoint scope."
+    },
+    "ConnectionMonitorEndpointScopeItem": {
+      "properties": {
+        "address": {
+          "type": "string",
+          "description": "The address of the endpoint item."
```

Comment:
Could you add more information into the description? What's kind of address? IPv4?

Reply:
@MyronFanQiu , done.